### DISCUSSION TITLE
सुस्वागतम! Welcome!

### DISCUSSION THREAD

**Discussion by sanketgarade** [mukta-strot/marathi-shabd#8] [Announcements]
## सुस्वागतम्

- प्रकल्पात सहभागी होण्यापूर्वी आपण इथे आपली ओळख करून देऊ शकता. 
- योगदान करायची प्रक्रिया `contributing` फाईल मध्ये समजून सांगितली आहे. 
- आपल्याला काही प्रश्न/कल्पना असल्यास त्यावर आपण इथे चर्चा देखील करू शकतो. 

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanketgarade** [comment]
### नमस्कार मंडळी! सर्वांचे स्वागत आहे!

Sanket here. I am an embedded systems engineer by profession.

The idea of this project has been going on in my head since past couple of months and the motivation of the larger cause of preserving our language even longer than that (may be a year or so).

**I am looking forward to working with you all who share the same enthusiasm and motivation.**

**Comment by rohit-u-s** [comment]
Namaskar _/\_
Rohit here. I am in animation industry.
trying to learn code, but procrastination keeps saving me!?
Ohh and I would love to see Sanskrit based Marathi.

**Reply by sanketgarade** [reply]
नमस्कार रोहित!
अनिमेशन क्षेत्रात काम करून कोडिंग मध्ये आवड?! सुंदर 😃 

ह्या रेपो मध्ये बरेच मुद्दे (issues) अजुन अपूर्ण आहेत. तुम्हाला हवं त्या मुद्द्यावर तुम्ही काम चालू करू शकता. काही मदत लागल्यास कळवा.🙏🏼

**Reply by sanketgarade** [reply]
आणि हो, संस्कृत मधून मराठीत नवीन शब्द जोडणे नक्कीच एक मजेशीर व उपयोगी गोष्ट होऊ शकेल !!
